### PR TITLE
Add option to show album art in the dock

### PR DESCRIPTION
**Pull request by matthewlloyd**
This is a feature I miss from Vox.

screen shot 2014-12-12 at 1 03 15 am
screen shot 2014-12-12 at 1 03 30 am

### CODE REVIEW

**Comment by Tape** [comment]
Just tested it out and it works well and looks great. I'd like to see this be accepted as a new feature.

File: radiant-player-mac/AppDelegate.m [hunk 1]
```diff
@@ -758,6 +769,30 @@ - (void)notifySong:(NSString *)title withArtist:(NSString *)artist album:(NSStri
...
+        }
+    }
+}
+
+- (void)downloadAlbumArt:(NSString *)art
+{
+    NSURL *url = [NSURL URLWithString:art];
+    currentArt = [[NSImage alloc] initWithContentsOfURL:url];
+    if ([defaults boolForKey:@"dock.show-art"] && currentArt)
+    {
```
Comment (gholts) [inline radiant-player-mac/AppDelegate.m]: I think this bracket needs to move up a line, judging by the other if-else blocks in this file

Comment (matthewlloyd) [inline radiant-player-mac/AppDelegate.m]: Done.

**Comment by gholts** [comment]
@kbhomes give the people what they want!

**Comment by kbhomes** [comment]
Thank you for this, it's a great feature! Merged, will be included soon.

**Comment by matthewlloyd** [comment]
Superb, thank you!!

**Comment by gholts** [comment]
Hooray!

**Pull request merged**